docs_src/security/Ch-Authenticating.md

```diff
@@ -0,0 +1,140 @@
+# Authenticating to EdgeX Microservices
+
+!!! edgey "EdgeX 3.0"
+    Microservice-level authentication is new for EdgeX 3.0.
+
+## Introduction
+
+Starting in EdgeX 3.0, when EdgeX is run in secure mode,
+EdgeX microservices require an authentication token before
+they will respond to requests issued over the REST API.
+(These changes are detailed in the
+[EdgeX microservice authentication ADR](../design/adr/security/0028-authentication.md)
+and were introduced to mitigate against certain threats
+that originate from behind the API gateway or have somehow bypassed the API gateway.)
+
+Prior to EdgeX 3.0, requests that originated remotely
+were authenticated at the API gateway via an HTTP `Authorization`
+header that contained a JWT bearer token.
+Internally-originated requests required no authentication.
+In EdgeX 3.0, the `Authorization` header is **additionally** checked
+at the microservice level on a per-route basis,
+where the majority of URL paths require authentication.
+
+## How to Make Authenticated EdgeX Calls
+
+In order to make an authenticated EdgeX service call,
+an EdgeX service must have a secret store token issued
+by the EdgeX secret store.
+
+The [Configuring Add-on Services](./Ch-Configuring-Add-On-Services.md)
+chapter contains details on what is required to
+enroll a new microservice into EdgeX,
+for the purpose of obtaining a secret store token.
+
+From this point forward, it is assumed that the calling code
+has a valid secret store token obtained using the above process.
+
+### Remotely
```

Comment:
### Remotely - API Gateway

Reply:
Fixed in https://github.com/edgexfoundry/edgex-docs/pull/981/commits/e9c009c3d03b0c99d0db1ba0554660b96b1f23f7